terraform/data.tf

```diff
@@ -0,0 +1 @@
+data "aws_availability_zones" "available" {}
```

Comment:
Don't you need to use the `state` filter to get the available ones? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones#by-state

Maybe this is supposed to bring all zones but the name of the resource is a bit misleading, right?

Reply:
I had understood that it would be [optional](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones#argument-reference) and by default they are all included :thinking:

Reply:
I thought you only wanted the ones with state "available", not all  the zones. Sorry for the confusion

Reply:
> I thought you only wanted the ones with state "available", not all the zones. Sorry for the confusion

Hmm now I see what you meant by the misleading title, I'll update it (maybe `all_zones`) :smile: